src/FlightMap/Widgets/CameraPageWidget.qml

```diff
@@ -42,6 +42,7 @@ Column {
     property real   _labelFieldWidth:       ScreenTools.defaultFontPixelWidth * 30
     property real   _editFieldWidth:        ScreenTools.defaultFontPixelWidth * 30
     property bool   _communicationLost:     activeVehicle ? activeVehicle.connectionLost : false
+    property bool   _streamingEnabled:      false //TODO: determine what it should be, VideoPageWidget.qml does QGroundControl.settingsManager.videoSettings.streamConfigured
```

Comment:
Looks like it came from here:
https://github.com/mavlink/qgroundcontrol/blob/master/src/FlightMap/Widgets/VideoPageWidget.qml#L37

This video section under settings was copied and pasted from `VideoPageWidget.qml`

https://github.com/mavlink/qgroundcontrol/blob/master/src/FlightMap/Widgets/CameraPageWidget.qml#L392-L434

Reply:
not sure I understand your message
you suggest I should use the same line as in your first link? 
I'm happy to copy it :)

Reply:
No (or maybe). I was just pointing out where it came from. I don't remember the order but I think the video widget came first (along with those video specific settings). When the other one was created, some of the video settings were brought along but it missed the `_streamingEnabled` definition.